### ISSUE TITLE
Missing set of aria-valuenow in p:fileupload while uploading

### ISSUE THREAD

**Issue by mrcasio** [primefaces/primefaces#11523]
### Describe the bug

During an upload, the width of the progress bar is updated, but the "aria-valuenow" attribute remains at zero throughout the upload.

A fix can be implemented in the file at https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/resources/META-INF/resources/primefaces/fileupload/2-fileupload.js#L349 by adding the following block of code to simultaneously update the width of the progress bar and the value of the attribute:

```javascript
if (file.row) {
    var fileuploadProgress = file.row.children(".ui-fileupload-progress").find("> .ui-progressbar");
    fileuploadProgress.attr("aria-valuenow", progress);
    fileuploadProgress.find("> .ui-progressbar-value").css({
        width: progress + "%",
        display: "block"
    });
 }
``` 

### Reproducer

1. Go to https://www.primefaces.org/showcase/ui/file/upload/single.xhtml.
2. Select a file to upload. The UI progress bar fills with: aria-valuemin="0", aria-valuemax="100", aria-valuenow="0".
3. Start the upload. The width of the progress bar is updated, but aria-valuenow remains at zero.

### Expected behavior

1. Go to https://www.primefaces.org/showcase/ui/file/upload/single.xhtml.
2. Select a file to upload. The UI progress bar fills with: aria-valuemin="0", aria-valuemax="100", aria-valuenow="0".
3. Start the upload. The width of the progress bar is updated, and aria-valuenow should be updated to the same value as the width of the progress bar.

### PrimeFaces edition

None

### PrimeFaces version

13.0.5

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

Jakartar 11

### Browser(s)

Chrome 121.0.6167.185

**Comment by melloware** [comment]
@mrcasio good catch and thanks for the fix. Testing it now.
